Content.Server/Mind/Mind.cs

```diff
@@ -62,31 +96,36 @@ public Mind(NetUserId userId)
         ///     The session ID of the player owning this mind.
         /// </summary>
         [ViewVariables]
+        [DataField("pssUserId")]
         public NetUserId? UserId { get; private set; }
 
         /// <summary>
         ///     The session ID of the original owner, if any.
         ///     May end up used for round-end information (as the owner may have abandoned Mind since)
         /// </summary>
         [ViewVariables]
-        public NetUserId OriginalOwnerUserId { get; }
+        [DataField("pssOriginalOwnerUserId")]
+        public NetUserId OriginalOwnerUserId { get; private set; }
 
         [ViewVariables]
         public bool IsVisitingEntity => VisitingEntity != null;
 
         [ViewVariables]
+        [DataField("pssVisitingEntity")]
```

Comment:
All of these data-fields should just follow the normal naming convention. I.e., jut use `visitingEntity`.

Reply:
Never mind, saw your comment in the PR description saying its just temporary.